### ISSUE TITLE
learnen authentic kernel

### ISSUE THREAD

**Issue by hkhelen** [qcf-568/OSTF#17]
Thank you for sharing your code,I have 2 questions about your work.：
1、You mentioned "During inference, the proposed DAF framework still works in an E2E manner with the learned authentic kernel, as shown in Figure 5 of the paper." in the appendix of your paper . But I can't find where is the learned authentic kernel in your code, can you tell me where it is?
2、I find that single center loss is used when training，is it Lfeat you mentioned in your paper？Or is there any difference between them?

**Comment by qcf-568** [comment]
<img width="1230" height="237" alt="Image" src="https://github.com/user-attachments/assets/25d9845a-d36d-458a-89a9-ff7063977f45" />

1. The readme.md file has pointed out the authentic kernel implementation.
2. Single center loss is used to learn the authentic kernel. The loss function itself has no difference, the difference is that: Single center loss paper does not explicitly use the feature difference. It inputs the feature itself to the final classifier while our paper inputs the {feature-authentic_kernel} to the final classfier.
